client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -170,6 +170,27 @@ const StatsLocations: React.FC< StatsModuleLocationsProps > = ( { query, summary
 		/>
 	);
 
+	const divisionsTooltip = (
+		<StatsInfoArea>
+			{ translate(
+				'Countries and their subdivisions are based on {{link}}ISO 3166{{/link}} standards.',
```

Comment:
The design contains an underscore in the `ISO_3166` link, but I've dropped it since the actual source uses spaces instead.